cmd/config.go

```diff
@@ -35,6 +35,11 @@ func setGlobalConfigPath(configPath string) {
 	customConfigPath = configPath
 }
 
+// Sets the global config path to empty string.
+func resetGlobalConfigPath() {
+	setGlobalConfigPath("")
+}
+
```

Comment:
I was sort confused between whether to move it to test-utils or not. Will move it to test-utils.